frontend/pem_to_jks.sh

```diff
@@ -0,0 +1,7 @@
+openssl pkcs12 -export -out eneCert.pkcs12 -in server.pem
```

Comment:
take a single server.pem file containing SSL certificate and private key, and generate whatever is needed by the python SSL context (just the certificate and private key separated) and by the java SSL context (which needs things stored in a password-protected keystore in JKS format)